internal/controller/drplacementcontrol_controller.go

```diff
@@ -152,6 +155,14 @@ func (r *DRPlacementControlReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return ctrl.Result{}, err
 	}
 
+	isSchedulingDisabled := isDRPCSchedulingDisabled(drpc, ramenConfig)
+	if isSchedulingDisabled {
```

Comment:
Any reason to add this temporary variable? the function name is descriptive so this does not add anything. It is helpful if we use the value later but if we don't avoiding more state is better.

Reply:
no particular reason, I will remove this.